README.md

```diff
@@ -121,6 +121,97 @@ get 'products' do
 end
 ```
 
+### Relational Extensions
+
+If you use either `ActiveRecord` or `Mongoid`, it is possible to represent that relationship using the `#relation` DSL method.
```

Comment:
The DSL works the same regardless if you use `Mongoid` or `ActiveRecord`; past the differences in how the models look the actual code for a representer would be the same for both. The extension will find the correct adapter to use for the model that you feed it at runtime (which is something I should mention in the docs!). The value add here IMO is the unified interface for relations and OOB support for popular libraries vs. having to do explicit boilerplate. 😄 

Maybe a better piece of documentation would be making custom adapter classes (e.g. for user supplied domain objects) like in this PR's description?